api/auth/index.ts

```diff
@@ -124,23 +132,30 @@ export async function oauth(req: EnhancedRequest, res: express.Response, next: e
             logger.warn('Auth token  expired need to log in again')
             doLogout(req, res, 401)
         } else {
-            let orgId
-            let details
-
+          const userDetails = await asyncReturnOrError(getUserDetails(accessToken), 'Cannot get user details', res, logger, false)
+          const isPrdAdminRole = havePrdAdminRole(userDetails)
+console.log ('isPRD ADMIN ROLE : ', isPrdAdminRole)
```

Comment:
Remove console.log. You can use logger.warn('string') or logger.info('') for useful information messages.